packages/sanity/src/core/form/studio/FormBuilderInputErrorBoundary.tsx

```diff
@@ -38,8 +42,8 @@ export function FormBuilderInputErrorBoundary(
  * when there are no errors.
  * @internal
  */
-function ErrorCard(props: {error: unknown; onRetry: () => void}) {
-  const {error, onRetry} = props
+function ErrorCard(props: {error: unknown; info?: React.ErrorInfo; onRetry: () => void}) {
```

Comment:
just double checking but is React.ErrorInfo available in the react 18 types as well?

Reply:
Yup!